src/examples/draggable/DraggableListDefault.tsx

```diff
@@ -19,12 +20,14 @@ const items = [
 ];
 
 const StyledHeading = styled(MD)`
+  transition: color 0.2s ease-in-out;
   margin-bottom: ${p => p.theme.space.xs};
+  color: ${p => getColor({ variable: 'foreground.default', theme: p.theme })};
```

Comment:
Ideally, this will utilize Garden's `<Span>` with a semantic `hue` prop when that functionality becomes available (currently WIP) in order to demonstrate first-class Garden usage. This is also a good reminder for the `Span` to receive a `transition` property along with the new `hue` functionality. With that in mind, I wonder if we should just remove the added `transition` CSS from this PR?

Reply:
Agreed re: removing transition if that comes with Span / typography components. 💯